edge/pkg/edged/pleg/generic.go

```diff
@@ -21,484 +21,29 @@ This file is derived from K8S Kubelet code with reduced set of methods
 package pleg
 
 import (
-	"fmt"
-	"net"
-	"sort"
 	"time"
 
-	"k8s.io/api/core/v1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/clock"
-	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/klog"
-	podutil "k8s.io/kubernetes/pkg/api/v1/pod"
-	v1qos "k8s.io/kubernetes/pkg/apis/core/v1/helper/qos"
 	kubecontainer "k8s.io/kubernetes/pkg/kubelet/container"
 	"k8s.io/kubernetes/pkg/kubelet/pleg"
-	"k8s.io/kubernetes/pkg/kubelet/prober"
-	"k8s.io/kubernetes/pkg/kubelet/status"
-	kubetypes "k8s.io/kubernetes/pkg/kubelet/types"
-
-	"github.com/kubeedge/kubeedge/edge/pkg/edged/podmanager"
 )
 
 //GenericLifecycle is object for pleg lifecycle
 type GenericLifecycle struct {
 	pleg.GenericPLEG
```

Comment:
delete `GenericLifecycle ` struct , and use `pleg.GenericPLEG` directly?

Reply:
Good idea, have done.